### ISSUE TITLE
[BUG] Error executing luv callback

### ISSUE THREAD

**Issue by SJPBP** [codota/tabnine-nvim#169]
When I try to open nvim with Tabnine installed I following error:

```
Error executing luv callback:
...local/share/nvim/lazy/tabnine-nvim/lua/tabnine/utils.lua:163: attempt to concatenate local 'chunk' (a nil value)
stack traceback:
        ...local/share/nvim/lazy/tabnine-nvim/lua/tabnine/utils.lua:163: in function <...local/share/nvim/lazy/tabnine-nvim/lua/tabnine/utils.lua:162>
```

I followed the instructions to install Tabine on Linux. I have `curl` and `unzip` in `Path`. I am using lazy to install Tabnine. 

I added 
```
require('tabnine').setup({
  disable_auto_comment=true,
  accept_keymap="<Tab>",
  dismiss_keymap = "<C-]>",
  debounce_ms = 800,
  suggestion_color = {gui = "#808080", cterm = 244},
  exclude_filetypes = {"TelescopePrompt", "NvimTree"},
  log_file_path = nil, -- absolute path to Tabnine log file
})
```

to `init.lua` and then I started to get the error described above.

I thought once Tabnine was installed, I could just use it.

**Version Info:**

- OS:
```
NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo
```
- Neovim version:
```
NVIM v0.9.5
Build type: Release
LuaJIT 2.1.1713773202
```
- Installed Tabnine Binaries (`ls -A /path/to/plugin/binaries`):
```
Error: chunk is nil 4.160.0
```

- Active Tabnine Binaries (`cat /path/to/plugin/binaries/.active`):
```
cat -- "$(nvim --headless -c 'lua io.stdout:write(vim.fn.stdpath("data"))' -c qa)"/*/tabnine-nvim/binaries/.active
Error: chunk is nilzsh: no matches found: /home/SJPBP/.local/share/nvim/*/tabnine-nvim/binaries/.active
```

**Comment by aarondill** [comment]
@amirbilu this was broken in #168 
It just needs the `if chunk then` check added back in

**Comment by amirbilu** [comment]
@aarondill feel free to add if you have time

**Comment by aarondill** [comment]
@amirbilu I've fixed this at #171
